docs/docsite/rst/reference_appendices/interpreter_discovery.rst

```diff
@@ -3,10 +3,13 @@
 Interpreter Discovery
 =====================
 
+Note: the behavior of this option changed in ansible-core 2.17. Consult the
+previous documentation if you are using ansible-core < 2.17.
```

Comment:
This note seems a little intrusive and unnecessary to me. Documentation changes with each release and version details are added to the changelog.

A better way to phrase this would be "consult the version of the documentation that corresponds to the version of Ansible that you're using" but that sounds too obvious.

Reply:
Just wanted to be friendly to the readers. But I agree you should read the version of the documentation of the version you are using.

I will remove it, but we should back-port these documentation changes to 2.17 and 2.18 I think.